src/controllers/auth.ts

```diff
@@ -43,24 +43,24 @@ export const loginUser = async (req: any, res: any) => {
 export const signupUser = async (req: any, res: any) => {
   console.log('Coming inside signup ====>');
   const { role, email, password, projects, preferences } = req.body;
-  const currentUserId = req.user.id; // Assuming user ID is attached to the request by auth middleware
-  const currentUserRole = req.user.role; // Assuming user role is attached to the request by auth middleware
+  // const currentUserId = req.user.id; // Assuming user ID is attached to the request by auth middleware
```

Comment:
this shouldnt be commented out. This was just supposed to be a temporary, local fix to enable us to create the first user